admin_manual/release_notes.rst

```diff
@@ -12,6 +13,61 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.4_release_notes_label:
+
+Changes in 10.0.4
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues of ownCloud Server 10.0.4 that need your attention:
```

Comment:
How about "known issues in ownCloud Server" instead of "known issues of ownCloud Server"?

Reply:
👍

Reply:
changed